### ISSUE TITLE
Too low accuracy in evaluation

### ISSUE THREAD

**Issue by fukui-kentaroh** [charlesq34/pointnet2#219]
When I run the code in Google Colab, the accuracy in training is over 90%, but the accuracy in evaluation is much lower. The output looks as follows.

```
 ---- batch: 550 ----
mean loss: 0.154208
accuracy: 0.943750
 ---- batch: 600 ----
mean loss: 0.160189
accuracy: 0.946250
2021-04-07 06:03:49.119485
---- EPOCH 250 EVALUATION ----
eval mean loss: 7361.237774
eval accuracy: 0.062399
eval avg class acc: 0.048500
```

Does anyone know the cause?
By the way, the version of Python in Colab is 3.7.10 and the version of Tensorflow is 2.4.1, so I made some changes to absorb the difference between the versions.

**Comment by pratibhashinde** [comment]
Hi,

I am also getting very low accuracy for evaluation. Did you manage to solve it? Can anyone help?

Thanks.

**Comment by elenacliu** [comment]
Are the labels in your dataset correct?

**Comment by lisj575** [comment]
Maybe you can check the batch_normalizaiton function you use
